src/table/sticky-columns/use-sticky-columns.ts

```diff
@@ -2,53 +2,32 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import React, { useCallback, useEffect, useMemo, useRef } from 'react';
-import AsyncStore from '../../area-chart/async-store';
+import AsyncStore, { useSelector } from '../../area-chart/async-store';
```

Comment:
This utility has not passed the formal team review: https://github.com/cloudscape-design/components/pull/998

This code is not allowed to change:

Reply:
ok, reverted the change related to the use of async store